src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -93,7 +99,35 @@ export interface SavedObjectsServiceSetup {
     id: string,
     factory: SavedObjectsClientWrapperFactory<KibanaRequest>
   ) => void;
+}
 
+/**
+ * @internal
+ */
+export type InternalSavedObjectsServiceSetup = SavedObjectsServiceSetup;
```

Comment:
Now these types are the same. I wasn't sure if I was supposed to remove `InternalSavedObjectsServiceSetup` or to  just keep the alias. For maintainability I'm leaning to keep it, as it would avoid re-introducing it if they diverge again later.